### PR TITLE
update System.Text.Encodings.Web version

### PR DESCRIPTION
**Pull request by epananth**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

Issue -> https://github.com/dotnet/core-eng/issues/14879

Allowed versions -> https://dev.azure.com/dnceng/internal/_componentGovernance/dotnet-arcade/alert/6087962?typeId=6283540

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -61,7 +61,7 @@
     <SystemReflectionMetadataVersion>1.6.0</SystemReflectionMetadataVersion>
     <SystemRuntimeCompilerServicesUnsafeVersion>4.7.0</SystemRuntimeCompilerServicesUnsafeVersion>
     <SystemRuntimeInteropServicesRuntimeInformation>4.3.0</SystemRuntimeInteropServicesRuntimeInformation>
-    <SystemTextEncodingsWebVersion>4.7.2</SystemTextEncodingsWebVersion>
+    <SystemTextEncodingsWebVersion>4.7.1</SystemTextEncodingsWebVersion>
```
Comment (riarenas) [inline eng/Versions.props:64]: This looks like a downgrade to me?

**Review by riarenas — CHANGES_REQUESTED** [review]
This is a version downgrade.

**Comment by riarenas** [comment]
I think what you want to do is find which project is bringing in 4.6.0, and update that one. Downgrading versions.props to use 4.7.1 does not seem right to me.

**Comment by epananth** [comment]
Closing this PR out. This isn't the correct way to resolve this

**Pull request closed without merging**